.scripts/typecheck/command.ts

```diff
@@ -0,0 +1,19 @@
+import { execSync } from 'child_process';
+import { executeCommand } from '../utils/exec';
+
+function typecheckCommand(args: string) {
+  const command = `npm run code:typecheck ${args}`;
+  return execSync(command, { encoding: 'utf-8' });
+}
+
+export function runTypecheck() {
+  try {
+    const logs = executeCommand(typecheckCommand);
+    console.log(logs);
```

Comment:
without it, does it still print to screen?

Reply:
No, it doesn't

Reply:
I think we don't need that, we only care about errors if it has any. So you can just print out like "Type looks good" with "check icon" 
```suggestion
    console.log(Checking type... 🩺 🔬 🔭');
    executeCommand(typecheckCommand);
    console.log(chalk.green('✔ Type looks good'));
```